javascriptv3/example_code/sns/README.md

```diff
@@ -1,58 +1,103 @@
-- [Create a topic](src/sns_createtopic.js)
-- [Delete a topic](src/sns_deletetopic.js)
-- [Get SMS type](src/sns_getsmstype.js)
-- [Get topic attributes](src/sns_gettopicattributes.js)
-- [List opted out numbers](src/sns_listnumbersoptedout.js)
-- [List subscriptions](src/sns_listsubscriptions.js)
-- [List topics](src/sns_listtopics.js)
-- [Publish SMS](src/sns_publishsms.js)
-- [Publish to topics](src/sns_publishtotopic.js)
-- [Set SMS type](src/sns_setsmstype.js)
-- [Set topic attributes](src/sns_settopicattributes.js)
-- [Subscribe to an app](src/sns_subscribeapp.js)
-- [Subscribe to an email](src/sns_subscribeemail.js)
-- [Subscribe to Lambda](src/sns_subscribelambda.js)
-- [Unscribe](src/sns_unsubscribe.js)
-
-**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
-[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-example-javascript-syntax.html).
-
-
-## Getting started
-
-1. Clone the [AWS SDK Code Samples repo](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment. See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for instructions.
-
-2. Install the dependencies listed in the package.json.
-
-**Note**: These include the client module for the AWS services required in these example, 
-which is "@aws-sdk/client-sns".
-```
-npm install node -g
-cd javascriptv3/example_code/sns
-npm install
-```
 
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `javascriptv3` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Check whether a phone number is opted out](libs/snsClient.js#L6) (`CheckIfPhoneNumberIsOptedOut`)
+* [Confirm an endpoint owner wants to receive messages](libs/snsClient.js#L6) (`ConfirmSubscription`)
+* [Create a topic](libs/snsClient.js#L6) (`CreateTopic`)
+* [Delete a subscription](libs/snsClient.js#L6) (`Unsubscribe`)
+* [Delete a topic](libs/snsClient.js#L6) (`DeleteTopic`)
+* [Get the properties of a topic](libs/snsClient.js#L6) (`GetTopicAttributes`)
+* [Get the settings for sending SMS messages](libs/snsClient.js#L6) (`GetSMSAttributes`)
+* [List the subscribers of a topic](libs/snsClient.js#L6) (`ListSubscriptions`)
+* [List topics](libs/snsClient.js#L6) (`ListTopics`)
+* [Publish to a topic](libs/snsClient.js#L6) (`Publish`)
+* [Set the default settings for sending SMS messages](libs/snsClient.js#L6) (`SetSmsAttributes`)
+* [Set topic attributes](libs/snsClient.js#L6) (`SetTopicAttributes`)
+* [Subscribe a Lambda function to a topic](libs/snsClient.js#L6) (`Subscribe`)
+* [Subscribe a mobile application to a topic](libs/snsClient.js#L6) (`Subscribe`)
+* [Subscribe an email address to a topic](libs/snsClient.js#L6) (`Subscribe`)
```

Comment:
Unfortunately, these all link to the same client import.

Reply:
Oof, that's write me 😬